docs/csharp/fundamentals/types/anonymous-types.md

```diff
@@ -61,6 +61,10 @@ You cannot declare a field, a property, an event, or the return type of a method
 
 Because the <xref:System.Object.Equals%2A> and <xref:System.Object.GetHashCode%2A> methods on anonymous types are defined in terms of the `Equals` and `GetHashCode` methods of the properties, two instances of the same anonymous type are equal only if all their properties are equal.
 
+> [!NOTE]
+> The [default accessibility level](../../programming-guide/classes-and-structs/access-modifiers.md#default-access-summary-table) of an anonymous type is `internal`, hence two anonymous types defined in different assemblies are not of the same type.
```

Comment:
Small nit: I'd remove "default" because there isn't a syntax to override it for an anonymous type.

```suggestion
> The [accessibility level](../../programming-guide/classes-and-structs/access-modifiers.md#default-access-summary-table) of an anonymous type is `internal`, hence two anonymous types defined in different assemblies are not of the same type.
```